pinakes/main/auth/views.py

```diff
@@ -42,9 +42,6 @@ def get_object(self):
 class SessionLogoutView(APIView):
     permission_classes = (IsAuthenticated,)
 
-    def get_serializer(self):
```

Comment:
This method was added to address a DRF error reported in console. We will still need a serializer. In stead of return a `None` serializer, we can remove this method, and add
```
from rest_framework.serializers import Serializer
serializer_class = Serializer
```
I have tested and prove it works.

BTW the 500 error occurs only when attempt to GET the logout endpoint which is not allowed.